docs/Zeta_environment_setup.md

```diff
@@ -206,4 +205,5 @@ TBD
 
 ### Problems
 
-- In the REST API of "add compute instance ports", what do "ips_port", "mac_port", "ip_node" and "mac_node" mean, respectively?
+- I checked DISABLED_2_ports_CREATE_test_traffic_PARENT in /test/gtest/aca_test_ovs_l2.cpp. There is a step to construct a neighbor port. Does our test also need to add a neighbor port?
+- In addition to issuing the goal state that created the port, do we also need to issue the goal state of the zeta information about this port?

```

Comment:
yes, see DISABLED_auxgateway_test under aca_test_zeta_programming.cpp on how the goal state message with Zeta information is constructed, look at how VPC state is contructed with Zeta gateway information.